src/views/mentorship/mentorship.tsx

```diff
@@ -1,12 +1,12 @@
-import { MentorshipHero } from './mentorship-hero/ui/mentorship-hero';
-import { DetailsMentorship } from './ui/details/details';
-import { MentorActivities } from './ui/mentor-activities';
-import { MentorshipCourses } from './ui/mentorship-courses/mentorship-courses';
-import { MentorsAfterRegister } from '@/views/mentorship/ui/mentors-after-register';
 import { Benefits } from '@/widgets/benefits';
 import { MentorsDocs } from '@/widgets/mentors-docs';
 import { MentorsFeedback } from '@/widgets/mentors-feedback';
 import { MentorsRegister } from '@/widgets/mentors-register';
+import { MentorActivities } from '@/widgets/mentorship-activities';
+import { MentorshipAfterRegister } from '@/widgets/mentorship-after-register';
+import { MentorshipCourses } from '@/widgets/mentorship-courses/mentorship-courses';
+import { MentorshipDetails } from '@/widgets/mentorship-details/details/mentorship-details';
```

Comment:
we can remove the extra nesting here.
```suggestion
import { MentorshipDetails } from '@/widgets/mentorship-details/mentorship-details';
```